pocket.go

```diff
@@ -17,101 +17,148 @@ var (
 func main() {
 	// Parse command line flags.
 	mode, sc, err := auxiliary.ParseArgs(os.Args)
-	if err != nil {
-		if err.Error() == "help" {
-			os.Exit(0)
-		} else {
-			fmt.Println(err)
-			os.Exit(1)
-		}
+	if err != nil && err.Error() != "help" {
+		fmt.Println(err)
+		return
```

Comment:
Yeah, maybe. Don't want to terminate anywhere to be fair, but since that's before any sensitive memory is allocated, it should be fine. Noted.